assets/js/common/ActivityLogsConfig/ActivityLogsConfig.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { noop } from 'lodash';
+
+import Button from '@common/Button';
+
+/**
+ * Simplistic utility to transform a time interval in a readable string
+ */
+function TimeLabel({ time }) {
+  if (typeof time === 'string') {
+    return time;
+  }
+  if (
+    typeof time === 'object' &&
+    time !== null &&
+    'value' in time &&
+    'unit' in time
+  ) {
+    return `${time.value} ${time.unit}${time.value > 1 ? 's' : ''}`;
+  }
+  return <em>invalid value</em>;
+}
+
+/**
+ * Render the saved configuration for the Activity Logs.
+ * Editing is demanded to another component, activated by the Edit button
+ *
+ * @param {{value: number, unit: "days"|"months"|"years"}} props.retentionTime A structured `{value,unit}`
```

Comment:
also `weeks`

(typescript intensifies :smile:)

Reply:
Fixed in https://github.com/trento-project/web/pull/2706/commits/f6c4315e173e09a5dc71cba162b6e3df8669ac18